src/diff/index.js

```diff
@@ -132,10 +132,9 @@ export function diff(parentDom, newVNode, oldVNode, context, isSvg, excessDomChi
 
 			c.base = newVNode._dom;
 
-			while (tmp=c._renderCallbacks.pop()) {
-				if (c._nextState) { c.state = c._nextState; }
-				tmp.call(c);
-			}
+			tmp = c._renderCallbacks;
+			c._renderCallbacks=[];
+			tmp.some(cb => { cb.call(c); });
```

Comment:
Should be safe to omit the curlies here, but won't change the ES5 output.

`tmp.some(cb => cb.call(c));`

Reply:
The ES5 output is different. `cb => cb.call(c)` becomes `function (cb) { return cb.call(c); }` whereas `cb => { cb.call(c); }` becomes `function (cb) { cb.call(c); }` ([babel repl](https://babeljs.io/repl#?babili=false&browsers=&build=&builtIns=false&spec=false&loose=false&code_lz=MYewdgzgLgBAhjAvDYAjJA-FqB0w4A2BAFMAJQBQoks6yamMA3tnoSeQNwwC-QA&debug=false&forceAllTransforms=false&shippedProposals=false&circleciRepo=&evaluate=true&fileSize=false&timeTravel=false&sourceType=module&lineWrap=true&presets=es2015%2Creact%2Cstage-2%2Ces2015-loose&prettier=true&targets=&version=7.6.4&externalPlugins=)).

I think we need to curlies so that `.some` doesn't bail out if the callback returns a truthy value [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some). Since these can be user provided callbacks we can't trust the return value.